fdbserver/storageserver.actor.cpp

```diff
@@ -799,6 +808,16 @@ struct StorageServer {
 		// Bytes of the mutations that have been added to the memory of the storage server. When the data is durable
 		// and cleared from the memory, we do not subtract it but add it to bytesDurable.
 		Counter bytesInput;
+		// Bytes pulled from TLogs, it counts the size of the key value pairs, without any overhead.
+		Counter logicalBytesInput;
+		// Bytes pulled from TLogs for moving-in shards.
+		Counter logicalBytesMoveInOverhead;
+		// Bytes committed to the underlying storage engine by SS, it counts the size of key value pairs.
+		Counter kvBytesCommit;
+		// Count of clearRange operatons to the storage engine.
+		Counter kvClearRange;
```

Comment:
naming convention:
For the number of operations, we use plural, such as kvClearRanges;
For the bytes of operations, we use bytes, such as kvClearRangeBytes;

Reply:
Thanks for the tip! Change applied.